packages/esm-form-engine-app/package.json

```diff
@@ -40,7 +40,7 @@
     "@carbon/react": "^1.12.0",
     "@openmrs/openmrs-form-engine-lib": "next",
     "lodash-es": "^4.17.21",
-    "react-error-boundary": "^4.0.11"
+    "react-error-boundary": "^4.0.13"
```

Comment:
For conformity with https://github.com/openmrs/openmrs-form-engine-lib/blob/fe966ae243c7e9e7db8ad40e650bc1f17ac9cf06/package.json#L36.